index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div>
```

Comment:
- [ ] For good HTML semantics, I suggest you use the `main` tag here which is naturally used to wrap the main content of the website,